src/test/java/com/jcabi/github/RtGistCommentTest.java

```diff
@@ -41,16 +41,28 @@
 import javax.json.Json;
 import org.hamcrest.MatcherAssert;
 import org.hamcrest.Matchers;
+import org.junit.Rule;
 import org.junit.Test;
 
 /**
  * Test case for {@link RtGistComment}.
  * @author Giang Le (giang@vn-smartsolutions.com)
  * @version $Id$
+ * @checkstyle ClassDataAbstractionCouplingCheck (150 lines)
  */
 public class RtGistCommentTest {
 
     /**
+     * The rule for skipping test if there's BindException.
+     * @todo Apply this rule to all other classes that use MkGrizzlyContainer
```

Comment:
@piddubnyi use this format
`@todo #1017:30min Apply this rule to all other classes`